lib/zhr_devs/submissions/aggregates/check.ex

```diff
@@ -52,7 +52,8 @@ defmodule ZhrDevs.Submissions.Aggregates.Check do
     %Events.SolutionCheckCompleted{
       solution_uuid: cmd.solution_uuid,
       task_uuid: cmd.task_uuid,
-      points: cmd.points
+      score: cmd.score,
+      is_baseline: cmd.is_baseline
```

Comment:
I've read through the rest of this commit, and I'm a bit confused about how does is_baseline will propagate all the way to the score sheet. As far as I understand the logic right now, there's one evaluation triggered per submission, which will only update the submissions records for the submission submitted.

For example, Jons submits a baseline submission, now the multiplayer table has a bunch of data of the built-in solvers plus  his submission marked as a baseline (if we have an endpoint to do so).

Then Alice submits her submission, now multiplayer table has a bunch of data, but no way to tell which one is the baseline.

Right?

Reply:
I don't know how `is_baseline` is calculated in the `zhr_bakery`, but I think that it should be calculated per run.
With this in mind, my guess is that the `baseline` is valid only for the last torunament run.

Does it answers your question?

Reply:
```json
{"gen_multiplayer_score":[{"hashed_id":"-HeoxYD4z_TXBdefTuKTpYxTX2VdHkC5R1WE2bxwfmA=","is_baseline":false,"norm":null,"score":{"points":0},"team":null},{"hashed_id":"ltX89lzN_vh2DwbuEMqHcqac39joTwdXOypA_ktMFGg=","is_baseline":true,"norm":null,"score":{"points":48},"team":null}]
```